### PR TITLE
Branch2 home task2 to Main

### PR DESCRIPTION
**Pull request by Anastasia-cod**
This pull request consist of the home_tasks from Lesson 2

**1 Task:**
Screenshot 2023-02-23 at 00 30 26

- I am not sure that I do correct when the devision operation - I'm trying to compare operand2 with != 0. 
But this is the best solution I came up with this night:)

I found 'catch' operation and [DivideByZeroException], but I didn’t fully understand how to add it to the task...

Screenshot 2023-02-23 at 00 19 48

- Also Is it correct to use 'if -else' operators inside 'switch'?

**Task 2**
Screenshot 2023-02-23 at 00 29 27

I have no questions about this task

**Task 3**
Screenshot 2023-02-23 at 00 28 43

Analogically task 2 - no questions

**Task 4**
Screenshot 2023-02-23 at 00 32 17

Did I understand correctly that the two options (for solving this task) mean operators if -else and switch?

### CODE REVIEW

File: VS 21 February/VS 21 February/Program.cs [hunk 1]
```diff
@@ -2,8 +2,186 @@
...
+            Console.WriteLine($"The entered number is even");
+        }
+        else
+        {
+            Console.WriteLine($"The entered number is odd");
+        }
+ 
+    }
+
+
```
Comment (VladimirVolkovVD) [inline VS 21 February/VS 21 February/Program.cs]: one empty line before and after methods

File: VS 21 February/VS 21 February/Program.cs [hunk 2]
```diff
@@ -2,8 +2,186 @@
...
+        switch (number % 2)
+        {
+            case 0:
+                Console.WriteLine($"The entered number is even");
+                break;
+            default:
+                Console.WriteLine($"The entered number is odd");
+                break;
+        }
+
```
Comment (VladimirVolkovVD) [inline VS 21 February/VS 21 February/Program.cs]: remove empty line

File: VS 21 February/VS 21 February/Program.cs [hunk 3]
```diff
@@ -2,8 +2,186 @@
...
+    }
+
+
+
```
Comment (VladimirVolkovVD) [inline VS 21 February/VS 21 February/Program.cs]: extra  empty lines

File: VS 21 February/VS 21 February/Program.cs [hunk 4]
```diff
@@ -2,8 +2,186 @@
...
+            Console.WriteLine($"Translate to English: {goodWeather_5}");
+        }
+        else
+        {
+            Console.WriteLine($"There is no translation for this word");
+        }
+    }
+
+
+
```
Comment (VladimirVolkovVD) [inline VS 21 February/VS 21 February/Program.cs]: same

File: VS 21 February/VS 21 February/Program.cs [hunk 5]
```diff
@@ -2,8 +2,186 @@
...
+        {
+            Console.WriteLine($"The entered number is between 51 and 100");
+        }
+        else
+        {
+            Console.WriteLine($"The entered number is not included in any of the existing ranges. Please enter integer number from 0 to 100");
+        }
+    }
+
+
```
Comment (VladimirVolkovVD) [inline VS 21 February/VS 21 February/Program.cs]: extra lines

Comment (VladimirVolkovVD) [reply VS 21 February/VS 21 February/Program.cs]: one empty line before and after methods

File: VS 21 February/VS 21 February/Program.cs [hunk 6]
```diff
@@ -2,8 +2,186 @@
...
+        string sign = Console.ReadLine();
+        int result;
+
+        switch(sign)
+        {
+            case "+":
+                result = operand1 + operand2;
+                Console.WriteLine($"addition operation: {operand1} {sign} {operand2} = {result}");
+                break;
+
```
Comment (VladimirVolkovVD) [inline VS 21 February/VS 21 February/Program.cs]: extra line

File: VS 21 February/VS 21 February/Program.cs [hunk 7]
```diff
@@ -2,8 +2,186 @@
 {
     static void Main(string[] args)
     {
-        string name = Console.ReadLine();
+        Lesson2_Task4_Way2();
+
+    }
+
+
```
Comment (VladimirVolkovVD) [inline VS 21 February/VS 21 February/Program.cs]: only one empty line before and after methods

**Review by VladimirVolkovD — CHANGES_REQUESTED** [review]
Tasks is good, need code style refactoding

File: VS 21 February/VS 21 February/Program.cs (hunk 5 above)
Comment (Anastasia-cod) [reply VS 21 February/VS 21 February/Program.cs]: Performed all required changes. Please, check.

**Review by VladimirVolkovD — APPROVED** [review]

**Pull request merged**